### ISSUE TITLE
Need a collaborative design space in tawk (e.g. iframe of Figma)

### ISSUE THREAD

**Issue by karth295** [invisible-college/tawk.space#23]

**Comment by karth295** [comment]
Sketchup doesn't have collaborative editing afaik, but it would be cool to be able to present something like this in tawk without screensharing. That way, other folks could play with a 3D model separate from the presenter.

https://www.sketchup.com/

**Comment by toomim** [comment]
Oh yeah!

What do you think about putting something like this in tawk, vs. putting tawk into something like that?

**Comment by karth295** [comment]
Fair enough. Google+ Hangouts seemed to have the same existential crisis several years ago. They started by trying to be center-of-the-universe and having integrations within Hangouts. But later seemed to shed all integrations, and just focus on the video piece.

To me it still seems more practical to have Tawk be the "collaborative platform" and have it stitch other applications together. Changing other websites is a harder problem :)

I also think to some extent, integrating video chat into websites is a solved problem. Effective collaboration on video chats is a more interesting problem imo.
